Common/TableProducer/fwdtrackToCollisionAssociator.cxx

```diff
@@ -45,8 +45,8 @@ struct FwdTrackToCollisionAssociation {
 
   void init(InitContext const&)
   {
-    if (doprocessFwdAssocWithTime == doprocessFwdStandardAssoc || doprocessMFTAssocWithTime == doprocessMFTStandardAssoc) {
-      LOGP(fatal, "Exactly one process function between standard and time-based association should be enabled!");
+    if ((int(doprocessFwdAssocWithTime) + int(doprocessFwdStandardAssoc) > 1) || (int(doprocessMFTAssocWithTime) + int(doprocessMFTStandardAssoc) > 1)) {
```

Comment:
What is the purpose of this change?

Reply:
I want to have the possibility to have oprocessFwdAssocWithTime == doprocessFwdStandardAssoc = false

Reply:
Exactly :)

Reply:
I see, then I recommend to separate the conditions and do `fatal` only when both fwd processes are enabled.

Reply:
Ok !

Reply:
You also need to make sure that at least on process function is enabled by these flags, or add a "dummy" `process` that does nothing.